lib/Models/WebMapServiceCatalogItem.ts

```diff
@@ -322,7 +322,19 @@ class GetCapabilitiesStratum extends LoadableStratum(
         {},
         this.capabilitiesLayers.get(this.catalogItem.layersArray[0])
       ) as any;
+
       if (out !== undefined) {
+        // The Dimension object is really weird and has a bunch of stray text in there
+        if ("Dimension" in out) {
+          const goodDimension: any = {};
+          Object.keys(out.Dimension).forEach((k: any) => {
```

Comment:
Tried a bunch of prettier ways to do this but it was being super fiddily so sorry it's ugly